### ISSUE TITLE
Prevent too-old versions of Cryptocat from even working

### ISSUE THREAD

**Issue by kaepora** [cryptocat/cryptocat#224]
What the title says. If you're below a certain known-okay version, the client just asks you to update and doesn't let you log in.

**Comment by andersbateva** [comment]
This simpler solution would make #66 less necessary by now.

**Comment by kaepora** [comment]
@andersbateva No, it wouldn't. Also, please stop commenting on every single issue with one-sentence comments. You're spamming my GitHub inbox.
